infrastructure/web3-adapter/src/evault/evault.ts

```
@@ -0,0 +1,205 @@
+import { GraphQLClient } from "graphql-request";
+import axios from "axios";
+import { v4 } from "uuid";
+
+export interface MetaEnvelope {
+    id?: string | null;
+    schemaId: string;
+    data: Record<string, any>;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)